app/src/main/java/com/sunlightlabs/congress/services/LegislatorService.java

```diff
@@ -140,4 +281,35 @@ private static Legislator legislatorFor(String url) throws CongressException {
 		return legislators;
 	}
 
+    // TODO: rename to legislatorsFor and remove old Sunlight method
+    private static List<Legislator> proPublicaLegislatorsFor(String url) throws CongressException {
+        List<Legislator> legislators = new ArrayList<Legislator>();
+        try {
+            JSONArray results = ProPublica.resultsFor(url);
+
+            // 'get specific member' puts members in 'results'.
+            // 'get current members by state/district' also uses 'results'.
+            // But other member responses use a subfield of "members".
+            // Need to introspect on the first result object to figure out
+            // if that's the case. Documented behavior here:
+            // https://github.com/propublica/congress-api-docs/issues/42
+
+            JSONArray members = results;
+            if (results.length() > 0) {
+                JSONObject firstResult = results.getJSONObject(0);
+                if (!firstResult.isNull("members"))
+                    members = firstResult.getJSONArray("members");
+            }
```

Comment:
The above code attempts to introspect on the contents of `results`, so that we can detect if we're getting a response where there's a sub-object of `members` (the state/district listing).